pipelines/mypy.nox.py

```diff
@@ -46,3 +46,13 @@ def generate_stubs(session: nox.Session) -> None:
 
 def _generate_stubs(session: nox.Session) -> None:
     session.run("stubgen", *STUBGEN_GENERATE, "-o", ".", "--include-private", "--no-import")
+
+    for path in STUBGEN_GENERATE:
+        # We add "i" because we turn ".py" into ".pyi"
+        stub_path = path + "i"
+
+        with open(stub_path, "r") as fp:
+            content = fp.read()
+
+        with open(stub_path, "w") as fp:
+            fp.write("# This file was automatically generated by `nox -s generate-stubs`. Do not update!\n\n" + content)
```

Comment:
Probably better to say something like "do not manually update/modify" here

Reply:
Updated :+1:

Reply:
You forgot to re-run the nox task

Reply:
Yep :sweat_smile: